com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -47,6 +49,11 @@ public enum ScriptableRenderPassInput
     [MovedFrom("UnityEngine.Rendering.LWRP")] public abstract partial class ScriptableRenderPass
     {
         public RenderPassEvent renderPassEvent { get; set; }
+        public DebugMaterialIndex debugMaterialIndex { get; set; }
```

Comment:
We need to discuss how to encapsulate this for debug, this was very hacky solution for hackweek only.